File: tools/dbench/answers/python_haiku/s2/f2_adults.py
lines = open("users.txt").readlines()
result = []
for line in lines:
    parts = line.strip().split()
    if len(parts) == 3:
        try:
            if int(parts[2]) > 25:
                result.append(parts[1])
        except ValueError:
            pass
print(" ".join(result))

File: tools/dbench/answers/rust_haiku/s3/f2_adults.rs
use std::fs;

fn main() {
    let txt = fs::read_to_string("users.txt").unwrap_or_default();
    let mut res = Vec::new();

    for line in txt.lines() {
        let parts: Vec<&str> = line.split_whitespace().collect();
        if parts.len() == 3 {
            if let Ok(age) = parts[2].parse::<i32>() {
                if age > 25 {
                    res.push(parts[1]);
                }
            }
        }
    }

    println!("{}", res.join(" "));
}
